src/collector.ts

```diff
@@ -132,6 +156,15 @@ class NaiveGomodParser {
             return dependencies;
         }, []);
 
+        goModDeps = goModDeps.map(dep => {
```

Comment:
This entire block should be moved after goImport check, Actually replacement should be done just before returning the dependencies. This will not work properly.